<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectDirDialog</class>
 <widget class="QDialog" name="selectDirDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select SubStacks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/imgs/sew1.png</normaloff>:/imgs/sew1.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="BaumWidget" name="listWidget">
     <property name="font">
      <font>
       <family>Hack</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pLorder">
       <property name="font">
        <font>
         <family>Hack</family>
        </font>
       </property>
       <property name="text">
        <string>slice order:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pCBorder">
       <property name="font">
        <font>
         <family>Hack</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>reverse</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pButton_ok">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Hack</family>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaumWidget</class>
   <extends>QListWidget</extends>
   <header>baumwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>selectDirDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
